### DISCUSSION TITLE
FIDI: Mapping data leads to TOO_MANY_REDIRECTS error

### DISCUSSION THREAD

**Discussion by baflo** [firefly-iii/firefly-iii#5480] [Firefly III]
Hi,

I've got a problem with the data mapping in FIDI: the import itself works well, but if I want to map data in the config step, it leads to a TOO_MANY_REDIRECTS error in the browser. The devtools network tab show that it will repeatedly call `/imports/convert` and `/imports/mapping` (or sth similar) for about 20 times, then the error occurs.

I cannot see any errors in the FIDI-logs, altough I set it to `APP_DEBUG=true` and `APP_LOG_LEVEL=debug`.

Does anybody have suggestions?

Cheers

**Comment by JC5** [comment]
https://github.com/firefly-iii/firefly-iii/issues/5463

**Comment by JC5** [comment]
Not really an answer I know, but I'm aware of the issue. You could help me and send me a CSV file (james@firefly-iii.org) and the settings you've checked / unchecked on the config page.

**Comment by baflo** [comment]
Thank you, I sent you a mail with the data. Just for completion, here are some details I also mentioned in the mail:

I've got a rather complicated setup for publishing my services to the web, could it be sth with that? I use dockerized version with Docker Compose and added nginx reverse proxies for both FF3 and FIDI to add all the forward headers stuff for protocol, host, etc. The nginx service themselves are published using an ngrok-like tool named "telebit".

telebit-Daemon (ff3.*.telebit.io) -> local port 8080 -> nginx (dockerized) -> ff3 (dockerized)
telebit-Daemon (fidi.*.telebit.io) -> local port 8081 -> nginx (dockerized) -> fidi (dockerized)
